src/test/scala/firrtl_interpreter/DriverSpec.scala

```diff
@@ -15,15 +16,15 @@ class DriverSpec extends FreeSpec with Matchers {
           |    output y : UInt<1>
           |    y <= x
         """.stripMargin
-      //      val interpreter = Driver.execute(Array.empty[String], input)
-      val interpreter = Driver.execute(Array("--fint-verbose"), input)
 
-      interpreter should not be empty
+      val source = FirrtlSourceAnnotation(input)
+      val result = Driver.execute(Array("--fint-verbose"), Seq(source))
```

Comment:
This appears to be causing:
```bash
Error: Unknown option --fint-verbose
Try --help for more information.

Failed to parse command line options
firrtl.FIRRTLException: Failed to parse command line options
	at firrtl.options.ExecutionOptionsManager$$anonfun$parse$1.apply(ExecutionOptionsManager.scala:69)
	at firrtl.options.ExecutionOptionsManager$$anonfun$parse$1.apply(ExecutionOptionsManager.scala:69)
	at scala.Option.getOrElse(Option.scala:121)
	at firrtl.options.ExecutionOptionsManager.parse(ExecutionOptionsManager.scala:69)
	at firrtl_interpreter.Driver$.execute(Driver.scala:33)
	at firrtl_interpreter.DriverSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply$mcV$sp(DriverSpec.scala:21)
	at firrtl_interpreter.DriverSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(DriverSpec.scala:10)
	at firrtl_interpreter.DriverSpec$$anonfun$1$$anonfun$apply$mcV$sp$1.apply(DriverSpec.scala:10)
 ...
```

Reply:
This PR appears to be missing the `META-INF` `ServiceLoader` registration... PR inbound. (There's a short option duplication after that that I'm looking into.)

Reply:
This referenced "short option duplication" is actually long option duplication. HEAD of https://github.com/freechipsproject/firrtl/pull/765 has an up to date version that shows the correct error.

Reply:
merged @seldridge PR that fixes this. Thanks!